salt/beacons/__init__.py

```diff
@@ -129,6 +129,25 @@ def add_beacon(self, name, beacon_data):
 
         return True
 
+    def modify_beacon(self, name, beacon_data):
+        '''
+        Modify a beacon item
```

Comment:
Does that make sense?  I'm happy to add the documentation in there as well if you think it's necessary.